db/scripts/init.sql

```diff
@@ -15,3 +15,23 @@ CREATE TABLE IF NOT EXISTS brothers(
 
 INSERT INTO brothers (pacificId, firstName, lastName, status, className, rollCall, email, phoneNumber, badStanding)
 VALUES (989123456, 'Nicolas', 'Ahn', 'Active', 'Chi', '000', 'na@gmail.com', '(123) 456-7890', 0);
+
+CREATE TABLE events(
+  eventID INT PRIMARY KEY NOT NULL, 
+  eventName TEXT NOT NULL,
+  category INT NOT NULL,
+  locations Text NOT NULL,
+  dates date NOT NULL
+);
+
+INSERT INTO events (eventID, eventName, category, locations, dates)
```

Comment:
A bit nitpicky, but `locations` and `dates` should be singular since an event tuple will only have a single location and single date.